mlos_bench/mlos_bench/tests/tunables/tunable_definition_test.py

```diff
@@ -0,0 +1,164 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Unit tests for checking tunable definition rules.
+"""
+
+import json
```

Comment:
Yeah, thought of that, but it's not strictly necessary in these cases since we're hand crafting the example json strings that we're loading from the source code without comments, but I'll make the change anyways for consistency.